### ISSUE TITLE
label problem

### ISSUE THREAD

**Issue by WowThankyou** [tkipf/gcn#56]
Hello everyone, 

I have some problem about label, can the nodes in our own dataset have multi-label? In other word, the node could belong to two or more classes.
If it can, could you please tell me how to implement !
Thank you very much!

**Comment by tkipf** [comment]
Yes, simply replace the loss function with a multi-label loss function (e.g. sigmoid cross entropy instead of softmax cross entropy).

**Comment by WowThankyou** [comment]
Thanks a lot!
